### ISSUE TITLE
Does dynamic_embedding support pre-trained embeddings?

### ISSUE THREAD

**Issue by treper** [tensorflow/recommenders-addons#212]
Can I load pre-trained embeddings and freeze them using dynamic embedding? How to do that? Thanks!

**Comment by rhdong** [comment]
Hi @treper , you mean warmup?

**Comment by treper** [comment]
> Hi @treper , you mean warmup?

I mean loading pre trained embedding to dynamic embedding, and then set some parameter like trainable=True to fine tune and trainable=False to freeze the loaded embedding.

**Comment by rhdong** [comment]
> > Hi @treper , you mean warmup?
> 
> I mean loading pre trained embedding(such as item graph embedding, pre trained text embedding ) to dynamic embedding, and then set some parameter like trainable=True to fine tune or set trainable=False to freeze the loaded embedding. The warmup you mentioned sounds like fine tune.

Okey, basically, I think you can borrow the [upsert](https://github.com/tensorflow/recommenders-addons/blob/master/docs/api_docs/tfra/dynamic_embedding/Variable.md#upsert) API to import pre-train embedding in any time of traing process, but maybe you had to write a logical to convert the source  format to TF tensors.

**Comment by rhdong** [comment]
Hi @treper， we have supported the warm start that should be helpful for implementing the pre-train features, please refer to [release note for r0.4.0](https://github.com/tensorflow/recommenders-addons/releases/tag/v0.4.0), and I will close the issue; please feel free reopen it if any help is needed. Thank you~
